README.md

```diff
@@ -60,7 +60,7 @@ Set up (the latest version of) [HAProxy](http://www.haproxy.org/) in Ubuntu syst
 * `haproxy_global_raw_options`: [default: `[]`]: Additional arbitrary lines to insert in the section
 
 * `haproxy_defaults_log`: [default: `global`]: Enable per-instance logging of events and traffic. `global` should be used when the instance's logging parameters are the same as the global ones. This is the most common usage
-* `haproxy_defaults_logformat`: [optional]: Allows you to customize the logs in http mode and tcp mode (e.g. `%{+Q}o\ %t\ %s\ %{-Q}r`)
+* `haproxy_defaults_logformat`: [optional]: Allows you to customize the logs in http mode and tcp mode (e.g. `"%{+Q}o\ %t\ %s\ %{-Q}r"`)
```

Comment:
Aren't you missing to `'`?

Reply:
@tersmitten What do you mean? Do you prefer `'` instead of `"`?

Reply:
To get it right it should be defined as:

```yml
haproxy_defaults_logformat: '"%{+Q}o\ %t\ %s\ %{-Q}r"'
```

Reply:
That's right, but the outer single quotes are not part of the value. Do you want to add them in the example?

Reply:
Yes, please